spring-boot-admin-server-ui/src/main/frontend/views/instances/loggers/logger-control.vue

```diff
@@ -63,7 +63,11 @@ export default {
   },
   methods: {
     hasEffectiveLevel(level) {
-      return this.value.some((l) => l.effectiveLevel === level);
+      return this.value.some((l) => {
+        return l.effectiveLevel === level || l.members
+          ? this.hasConfiguredLevel(level)
+          : false;
```

Comment:
as already discussed in the chat, I still think that this could be written in a easier-to-understand way